src/Guzzle/HmacAuthMiddleware.php

```diff
@@ -56,15 +56,18 @@ public function __invoke(callable $handler)
 
             $promise = function (ResponseInterface $response) use ($request) {
 
-                $authenticator = new ResponseAuthenticator($request, $this->key);
+                if($response->getStatusCode() != 401 ){
```

Comment:
snp: psr-2 = one space after if / before opening paren, and one space after cosing paren / before opening brace